README.md

```diff
@@ -37,6 +37,35 @@ cat `127.0.0.1 discovery-service.com
 ### Maybe you also need to initialize the cluster
 `docker exec -it db-1.com ./cockroach init --insecure`
 
+## Kubernetes installation with 1 node (minikube)
+1) install hypervisor (virtualbox or hyperkit)
+2) install minikube
+3) install kubectl
+4) `minikube config set driver virtualbox` or hyperkit
+5) `minikube start --container-runtime=cri-o` for local development better use docker instead of cri-o
```

Comment:
Почему for local development better use docker instead of cri-o ?

Reply:
что бы локально подключиться к docker daemon внутри кубера нужно просто сделать eval $(minikube docker-env). Вроде с cri-o так нельзя. + Форвардить сервисы с кубера на локальный порт можно только с docker runtime

Reply:
```suggestion
5) `minikube start --container-runtime=cri-o` for local development better use docker instead of cri-o because you can do `eval $(minikube docker-env)`
```

Reply:
done